### PR TITLE
Fix: Not working in the browser anymore

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

File: locales/core.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if (require && module) {
+if (typeof require != 'undefined' && module) {
```
Comment (ypid) [inline locales/core.js]: Please be more explicit: ` typeof require === 'function'`.

**Comment by ypid** [comment]
Thanks. Which browser is effected? I am testing with Firefox/Iceweasel and Chromium.

**Comment by amenk** [comment]
Chrome...

**Comment by amenk** [comment]
Or does the project require require-js or something? But I think no browser has `require` included.

**Comment by ypid** [comment]
Probably related to #98. It is all bundled with browserify now.

**Comment by amenk** [comment]
Is browserify a requirement now? Or is manul inclusion still supported? I am having a hard time upgrading my project there ... :-/

**Comment by ypid** [comment]
Should be quite simple. I was also not used to browserify. See https://github.com/ypid/opening_hours.js/blob/87acd8fefa28227660eaee71c144980759ad7c54/index.html#L9-L17

**Comment by amenk** [comment]
But I am packing everything together using gulp so I am not sure if I want to use browserify...

**Comment by ypid** [comment]
gulp seems interesting but I don’t use it yet. If you find a solution please let me know. I will let this PR open for a couple of days so you can push to patch-3 and tell me when you are done. Sounds good?

**Comment by amenk** [comment]
I think I am done with the patch-3 .. just added your suggestion.
What I do in gulp (actually the gulp module laravel-elixir) is merging the different JS file from my project in to one minified one.

**Comment by ypid** [comment]
All right.

> What I do in gulp (actually the gulp module laravel-elixir) is merging the different JS file from my project in to one minified one.

Was there a need for a workaround to make this work with opening_hours.js?

**Comment by amenk** [comment]
No. But currently I have issues with the translations for the opening-times-demo...

**Comment by ypid** [comment]
All right. When you have a workaround can you describe it in the README maybe?

**Comment by amenk** [comment]
I am using parts of the demo, does not really belong to the module.

**Comment by ypid** [comment]
I meant opening_hours.js + translations + gulp. I don’t know gulp yet so if you have something which is specific to opening_hours.js when using gulp I would be interested. Anyway, thanks for the patch.

**Comment by amenk** [comment]
no there is nothing special. It is pretty much only concatenating the .js files ... and minifying if you want it to.

**Comment by ypid** [comment]
OK, thanks for the hint. Sounds good.

**Comment by amenk** [comment]
While I am not sure, whether the lang-file loading for multiple resource files I did a while ago at https://github.com/ypid/opening_hours.js/blob/master/locales/core.js#L167 is a good approach.

**Comment by ypid** [comment]
I am also not sure about this. It is probably not best practice but it works for now. I am open to PRs if you find something better :wink:

**Pull request merged**